pkgs/development/rocm-modules/6/llvm/default.nix

```diff
@@ -213,12 +213,7 @@ let
         echo 'export CXX=clang++' >> $out/nix-support/setup-hook
       '';
   # Removes patches which either aren't desired, or don't apply against ROCm LLVM
-  removeInapplicablePatches =
-    x:
-    (
-      (lib.strings.hasSuffix "add-nostdlibinc-flag.patch" (baseNameOf x))
-      || (lib.strings.hasSuffix "clang-at-least-16-LLVMgold-path.patch" (baseNameOf x))
-    );
+  removeInapplicablePatches = x: (lib.strings.hasSuffix "add-nostdlibinc-flag.patch" (baseNameOf x));
```

Comment:
This patch no longer exists; it was moved to the wrapper.

Reply:
Nice!

Reply:
Dropped `removeInapplicablePatches`